File: labyrinth.py
#!/usr/bin/env python
# labyrinth.js translated to python

import math, random

for line in range(1, 60):
  for i in range(1, 29):
    s = "╱" if (math.floor((
      random.random() * 2) % 2)) else "╲"
    print(s, end="")

File: labyrinth.js
for (var line=1; line<60; line++) {
    for(var i=1;i<29;i++) {
        var s = (Math.floor((Math.random()*2)%2)) ? "╱" : "╲";
        process.stdout.write(s);
    }
    process.stdout.write('\n');
}
